standard/expressions.md

```diff
@@ -968,15 +968,13 @@ For a function member that includes a parameter array, if the function member is
   - for a fixed value parameter or a value parameter created by the expansion, an implicit conversion ([§10.2](conversions.md#102-implicit-conversions)) exists from the argument expression to the type of the corresponding parameter, or
   - for a `ref` or `out` parameter, the type of the argument expression is identical to the type of the corresponding parameter.
 
-In the case of a method group
+Additional rules determine whether a method is applicable or not based on the context of the expression:
 
-- If the group contains both instance and static members, instance members invoked without an instance receiver or context are not applicable.
-- If the group contains both instance and static members, static members invoked with an instance receiver are not applicable.
-- When there is no receiver, static members in a static context are applicable; otherwise, both static and instance members are applicable.
+- An instance method is not applicable when the invocation has no instance receiver and `this` cannot be used explicitly (such as in static methods, field initializers and constructor initializers).
```

Comment:
I don't know "instance receiver" is used elsewhere. I think what you're looking for is "when the receiver is not a variable or value", and on the next line "when the receiver is not a type" – this phraseology comes from §11.6.5.

Reply:
I'll go with that, but it needs to also take into account the possibility of there not being a receiver.

Reply:
Not sure on this, actually - no explicit receiver? I have yet to find a good canonical definition of "receiver".